main.js

```diff
@@ -0,0 +1,39 @@
+
+
+
+
+const tweeter = Tweeter()
+const renderer = Renderer()
+renderer.renderPosts(tweeter.getPosts())
+
+function delet_post(delet_buttom)
```

Comment:
In JS we do not use the `_` to make variables. We camelCase. So instead write it `deleteButton`